Extras/HistoryView/UIKit/COTrackViewController.m

```diff
@@ -107,7 +107,7 @@ - (UIColor *)suggestedColorForNode: (id <COTrackNode>)node
 
 - (UITableViewCell *)makeCellForNode: (id <COTrackNode>)node
 {
-    static NSString *TrackCellIdentifier = @"TrackCell";
+    static const NSString *const TrackCellIdentifier = @"TrackCell";
```

Comment:
I'm not sure, @qmathe do we want to use `static const NSString *const TrackCellIdentifier = @"...";`?

`const` is rarely used with ObjC objects in general, since there is no distinction in the type system between mutating and non-mutating ObjC method calls, like there is in C++.